ramls/request-move.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request to move a request from one item to another",
+  "properties": {
+    "destinationItemId": {
+      "description": "UUID of the item the request should be moved to",
+      "type": "string"
+    },
+    "requestType": {
```

Comment:
Is request type not required because the current type should be retained if it isn't provided?

Reply:
Correct.